src/SIL.LCModel.Core/SpellChecking/SpellEngine.cs

```diff
@@ -72,7 +72,7 @@ internal SpellEngine(string affixPath, string dictPath, string exceptionPath)
 #else
 				m_hunspellHandle?.Dispose();
 #endif
-				throw new Exception(e.Message);
+				throw;
```

Comment:
I just refreshed my memory - to re-throw the exception just `throw` is the preferred way because it preserves the original call stack (https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/throw#re-throwing-an-exception)